django_project/feti/models/campus.py

```diff
@@ -50,7 +50,7 @@ class Campus(models.Model):
     class Meta:
         app_label = 'feti'
         ordering = ['campus']
-        verbose_name = 'Provider'
```

Comment:
Let's leave this as provider. There was a big debate about this the last round, but it turns out `Provider` is the most correct name here.